.github/workflows/logging.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     timeout-minutes: 20
     defaults:
       run:
-        working-directory: ./dart
+        working-directory: ./logging
```

Comment:
thats a good catch hehe

Reply:
Yeah, I found it because PANA didn't warn us for the `dart:io` import. So there had to be a mistake in there.